### DISCUSSION TITLE
Is there a way to customise the primary theme color - 2176AE

### DISCUSSION THREAD

**Discussion by raghuwarrier** [thomvaill/log4brains#88] [Q&A] [answered]
I am looking forward to using this tool locally. One of the requests I received was to customise the color theme. Is there an easy way to do this? Thanks.

**Comment by thomvaill** [comment, marked as answer]
Hello @raghuwarrier,
This is not yet implemented.
Please create a feature request issue, describing precisely what you have in mind ;)
